drivers/iommu/amd/init.c

```diff
@@ -3348,16 +3348,14 @@ int __init amd_iommu_enable(void)
 	if (ret)
 		return ret;
 
-#if IS_ENABLED(CONFIG_SECURE_LAUNCH)
```

Comment:
Why is this removed? Won't it cause build failures with this option disabled?

Reply:
Not sure why 32bit would change the outcome, but reverted that part: https://github.com/TrenchBoot/linux/compare/257171792711fad05a8477be3beaf2cebff0830a..63ed6bb791786eab4b96d8510cbd23d4d87d356e